### DISCUSSION TITLE
Translating "Ezra Bible App"

### DISCUSSION THREAD

**Discussion by zhuiks** [ezra-bible-app/ezra-bible-app#321] [General]
I want to raise a question of whether to translate project title **Ezra Bible App** or not.

Somebody suggested that we should keep it to have a "brand name".

But there are two issues: 
1. **Ezra** has different pronunciation in different languages. I.e. in Ukrainian and Russian there is a clear "d": _Єздра /Yezdra/_. So the English name Ezra doesn't ring a bell for people who speak only Russian or Ukrainian.
2. **App** I think currently most of the users would understand this word no matter what language they use. But how "slangish" it is and either it used only for mobile apps or mobile and desktop apps depends on the language. Again as an example in Russian or Ukrainian there is no native word with the same root (Application will be _Приложение /prilojenie/_ in Russian or _Додаток /dodatok/_ or _Застосунок /zastosunok/_ in Ukrainian).

If we think that our users love the Word and the words, I think it's an important issue. But we should have a unified approach.
So dear translators' community what do you think?
@augustin-colesnic @reyespinosa1996 @lemtom @Rodolinux @LAfricain @MartinIIOT
@tobias-klein @alerque

**Comment by MartinIIoT** [comment]
I think for Slovakia: "Ezra Bible" is absolute OK without "App" word.

**Comment by augustin-colesnic** [comment]
Hi @zhuiks,

Well, Romanians are very savvy  people :innocent: ... at least the ones that know how to use a computer / tablet. 
So, I guess the term  **Ezra Bible App** is ok for us like the term **week-end** already is in our language. It will eventually catch on the users. We will focus to make the app useful enough  and the name will stick no matter what. :pray:

**Comment by tobias-klein** [comment]
In *German* it could actually be *Esra Bibel App*.

**Comment by reyespinosa1996** [comment]
I think Ezra Bible App could be rebranded into something more easy to translate, like maybe Ezra Bible Notebook wish could be easily translated to Spanish like "Cuaderno Bíblico Esdras" I don't know if this make any sense in Russian nor Ukranian.

**Reply by augustin-colesnic** [reply]
If it is to be turned into a study Bible i would go for '**Ezra Study Bible**'.  Indeed the word App is kind of implicit. If we look around what other apps have we can observe diff approaches: 
e.g. - [Logos](https://www.logos.com/).  [You Version](https://www.youversion.com/the-bible-app/). [MyBible](https://mybible.ro/)

**Comment by zhuiks** [comment]
Thanks everybody for expressing your opinions.

We just came out from rebranding from Ezra Project to Ezra Bible App. I think this name is good in terms of uniqueness and easy to remember. It's a bit long. So users would probably will shorten it in their day-to-day use to "Ezra Bible" or "Ezra" (i.e. "...So today I added the note to John 3 in **Ezra**)

Regarding translation. I realized that there are actually two very different occurrences. 
1. **External**: app title itself (in the title bar). That probably should always stay "Ezra Bible App" for all the languages to easily connect it with play-store, website and github references. (that's our brand!)
2. **Internal**: Mentions of the Ezra Bible App inside locale files (that was my initial thought when I started the topic). Here is one example:
![Снимок экрана от 2021-06-22 15-38-42](https://user-images.githubusercontent.com/3584119/122996417-7523a400-d370-11eb-8b7e-2b7fc21b92f3.png)

In my opinion, in this case the entire "Ezra Bible App" can be translated to additionally emphasize the meaning of the title.

Maybe under "Application Info" there could be a little description of the app, and we can connect the app title and its translated variant in that description.

**Reply by tobias-klein** [reply]
> Maybe under "Application Info" there could be a little description of the app, and we can connect the app title and its translated variant in that description.

I think that's generally a good idea, independent of whether we translate the app name or not. In the application info dialog we could add a purpose statement and a little bit of the history of the app. I just added issue #425.
